app/javascript/components/PageTitleWithContainer.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import PageTitle from 'components/PageTitle';
+
+const PageTitleWithContainer = ({ text }) => {
```

Comment:
This component implements a container for the page title so that it is properly spaced at the top of the page. Right now the Jobs page is the only page that uses this, but it can be added to any page that has the title component at the very top (every page except home and jobs authenticate).